bin/restydoc-index

```diff
@@ -357,3 +360,15 @@ sub gen_aliases {
 
     return @aliases;
 }
+
+sub shell_quote ($) {
+    my $ret = shift;
+
+    if (!defined $ret or $ret eq '') {
+        return "''";
+    }
+
+    $ret =~ s/([#&;`'"|*?~!<>^\(\)\[\]\{\}\$\\, ])/\\$1/g;
```

Comment:
Is this list exhausted? How about tabs and other control characters?

Reply:
You don't need to escape `(`, `)`, `{`, and `}` inside a character class (i.e., `[...]`).